### ISSUE TITLE
Transform Error when using unocss.config.ts as configuration file

### ISSUE THREAD

**Issue by JSFTI** [unocss/unocss#1017]
When using unocss.config.ts as configuration file, `npm run dev` returns error.
![2022_05_29_0pi_Kleki](https://user-images.githubusercontent.com/58502946/170858961-00007b36-3c94-4f28-8b54-d260b04b51ec.png)
Dependencies
![image](https://user-images.githubusercontent.com/58502946/170859028-516dec31-2b5c-4097-ab8d-474c14793d69.png)
vite.config.ts
![image](https://user-images.githubusercontent.com/58502946/170859058-eecede26-546f-4386-87fd-42f0e88ccea0.png)
unocss.config.ts
![image](https://user-images.githubusercontent.com/58502946/170859076-68cac211-b9ab-4e68-8029-b0ad554590cf.png)

However, if unocss.config.ts is removed, `npm run dev` runs successfully. Am I missing something?

**Comment by antfu** [comment]
Can you share a minimal reproduction? Thanks.

**Comment by Sfinx** [comment]
Same here (quasar2 + vite). Error appears from unocss v0.35.0. Reverting back to v0.34.1 eliminates the error

```
<path to app>/node_modules/@unocss/transformer-directives/dist/index.mjs:2
var _cssTree = require("css-tree");
               ^

Error: TRANSFORM_ERROR: Export namespace should be first transformed by `@babel/plugin-proposal-export-namespace-from`.
 
 <path to app>/node_modules/css-tree/lib/index.js:0:0
    at <path to app>/node_modules/@unocss/transformer-directives/dist/index.mjs:2:16
    at jiti (<path to app>/node_modules/jiti/dist/jiti.js:1:196506)
    at <path to app>/node_modules/unocss/dist/index.mjs:10:53
    at jiti (<path to app>/node_modules/jiti/dist/jiti.js:1:196506)
    at <path to app>/uno.config.js:1:177
    at jiti (<path to app>/node_modules/jiti/dist/jiti.js:1:196506)
    at loadConfigFile (<path to app>/node_modules/unconfig/dist/index.cjs:149:11)
    at async Object.load (<path to app>/node_modules/unconfig/dist/index.cjs:82:24)
    at async Object.loadConfig (<path to app>/node_modules/@unocss/config/dist/index.cjs:50:18)
    at async reloadConfig (<path to app>/node_modules/@unocss/vite/dist/index.cjs:67:20)
```

**Comment by JSFTI** [comment]
Gladly,
First, I create a new Vite project by `pnpm create vite`, and choose react as the framework and react-ts as variant
![image](https://user-images.githubusercontent.com/58502946/170859523-af5eb7a4-a15d-4b29-8922-8fc9b2422413.png)
Changed active directory to vite-project and run `pnpm i`
![image](https://user-images.githubusercontent.com/58502946/170859610-055d9fec-f40b-4a24-97e9-2a23b6575366.png)
Installs unocss with `pnpm i unocss`
![image](https://user-images.githubusercontent.com/58502946/170859638-dda5c025-6db2-43a8-99be-d37c40d3b906.png)
Import and add `Unocss()` to vite.config.ts plugin array.
![image](https://user-images.githubusercontent.com/58502946/170859664-e16ae755-1057-4af0-b058-25bab67d3ce3.png)
Create unocss.config.ts file
![image](https://user-images.githubusercontent.com/58502946/170859682-ab90952a-0480-4031-a601-49cda67036b8.png)
This is the current directory structure
![image](https://user-images.githubusercontent.com/58502946/170859691-a2c50c63-4807-4b4e-a460-9b88499fee50.png)
Run `npm run dev` to start development server
![image](https://user-images.githubusercontent.com/58502946/170859730-a3580f3c-1340-45d1-a24b-6b3ae1ea07a2.png)
Error occurs.

However, if unocss.config.ts is removed, no error occurs and unocss will work as intended.

**Comment by JSFTI** [comment]
Forgot to mention that uno.css is imported in main.tsx

**Comment by QiroNT** [comment]
This was already fixed by #1016, just it's not released yet.
Seems like there is no way to link a closed pull request.

**Comment by JSFTI** [comment]
I see, I guess the best course of action is to revert to 0.34.1 as mention by Sfinx. Thank you for your help.

**Comment by antfu** [comment]
Published v0.35.2

**Comment by Sfinx** [comment]
Thanks, works !
